infra/terraform/test-org/org/external_data/bigquery-external-table-test.csv

```diff
@@ -1,2 +1,2 @@
 column_a,column_b
-foo,bar
\ No newline at end of file
+foo,bar

```

Comment:
Likely unimportant, but this will force a replace of `google_storage_bucket_object.ci_bq_external_csv_file`